storage/rocksdb/ha_rocksdb.cc

```diff
@@ -5608,11 +5608,9 @@ int ha_rocksdb::read_range_first(const key_range *const start_key,
       DBUG_RETURN(HA_ERR_LOCK_DEADLOCK);
     }
 
-    MYSQL_TABLE_IO_WAIT(m_psi, PSI_TABLE_FETCH_ROW, active_index, 0, {
-      result =
-          index_read_map_impl(table->record[0], start_key->key,
-                              start_key->keypart_map, start_key->flag, end_key);
-    })
+    result =
```

Comment:
Yes, first, I believe it is inappropriate here and second, it is not 'visible' outside of sql/handler.cc as noted in commit message.